src/content/guides/farmlands-portal-introduction.mdx

```diff
@@ -0,0 +1,200 @@
+
+    <img src="/farmlands-payment-scan-barcode.png" alt="Scan and Enter Farmlands barcode on the Centrapay business portal" style=""/>
+
+4. Cardholder Not Present - Tick the box if you are seeking an authorisation for a sale where the Card is not physically present e.g. a phone or online order when a Card number has been quoted.
+
+    <img src="/farmlands-payment-select-card-holder-not-present.png" alt="Select card holder not present on the Centrapay business portal" style=""/>
+
+5. Click Create
+6. Check ID - This response may be given for high-value or high-risk transactions. The user is required to request the customer for photo ID to check against the Cardholder’s name. The transaction remains in a “pending” state until the user confirms they have authenticated the Card.
+
+    <img src="/farmlands-payment-check-photo-id.png" alt="Check photo ID and accept condition on the Centrapay business portal" style=""/>
+
+7. Approved - Response received.
+
+    <img src="/farmlands-payment-authorisation-approved.png" alt="Payment Authorization approved on the Centrapay business portal" style=""/>
+
+8. Copy Short Code - Once approved this will bring up the Authorisation Approved screen, showing the “Short Code” This needs to be copied to your invoices into the PO field or your usual field you would place the 9-digit Card Number.
+9. Click Close - This will redirect you to the Payment screen showing the authorisation history.
+
+    <img src="/farmlands-payment-payments-summary.png" alt="Summary of payments on the Centrapay business portal" style=""/>
+
+### **Invoicing & Payment**
+
+In order to complete settlement of the transaction a full GST invoice must still be submitted vendor.invoices@farmlands.co.nz or through your existing invoicing method (PDF Invoice or EDI).  The Short Code displayed on the Approval screen must be clearly displayed on your invoice.
+
+Farmlands require this “approved” Short Code to be shown on your invoice sent for payment. The Short Code should be copied (by clicking on the copy button shown) and pasted into the PO field or the current field where you would normally insert the Farmlands Card Number.
+
+<img src="/farmlands-payment-authorisation-approved.png" alt="Payment Authorization approved on the Centrapay business portal" style=""/>
+
+Using the authorisation portal provides more assurance for our Card Partners to receive payment on time. Invoicing terms and conditions still apply e.g. invoices are to be in on the first working day before 11.59 pm. Farmlands encourages invoices to be submitted daily rather than all at the end of the month. This allows Farmlands to apply the purchase to the customer’s account in a timely manner.
+
+Invoices must include the “short code” on every invoice. This should be in the same field for every invoice. The Farmlands Card Number can be provided as an addition to the short code in another field.
+
+> **Instore Transactions** <br />
```

Comment:
Not a fan of putting these `<br />` elements in our markdown 🤔 

Do we need a header on these callouts at all? Or does tabbing the content in 1 or 2 keep the body within the callout?

Reply:
No that doesn't seem to keep the content inside the callout

Reply:
Damn.

We could move these callouts under the correct `Create a Farmlands Authorisation` steps above so that we don't need the headers at all. Happy for this to be considered at a later time though - let's publish this